gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/jCard.ts

```diff
@@ -0,0 +1,21 @@
+import { buildCardMetadata } from "./types/card-metadata.factory";
+import { CardMetadata } from "./types/card-metadata.interface";
+
+class Card {
```

Comment:
jCard 2.0.... apparently it's a crap ton of "how do i set metadata for this", "how do I render a card" and some helper functions. With all of that refactored out we can turn this into just a sweet sweet POCO! (well, POTO or POJO i guess)